### PR TITLE
Allow ntuples to be usable for NMSSM analysis:

### PR DESCRIPTION
**Pull request by janekbechtel**
Add nmssm filelist and store additional quantities and producer to read out di-b jet quantities
This PR would also delete the git lfs files from before the re-skim (afaik we don't use them anymore), and in general remove git lfs as I think we did not account for the size limit (correct me if I am wrong)

### CODE REVIEW

**Review by janekbechtel — COMMENTED** [review]
now ready for PR

**Review by harrypuuter — APPROVED** [review]
Seems good to me, maybe one check, if SM still works with the changed Producer ordering (which is hard to do with the currently broken branch for DeepTau)

File: python/data/ArtusConfigs/Run2LegacyAnalysis_base.py [hunk 1]
```diff
@@ -32,7 +31,8 @@ def build_config(nickname, **kwargs):
   isDY = re.search("DY.?JetsToLLM(10to50|50)", nickname)
   isWjets = re.search("W.?JetsToLNu", nickname)
   isSUSYggH = re.search("SUSYGluGluToHToTauTau", nickname)
-  isSignal = re.search("HToTauTau",nickname)
+  isSignal = re.search("HToTauTau|125tautau",nickname)
```
Comment (swozniewski) [inline python/data/ArtusConfigs/Run2LegacyAnalysis_base.py]: Ist es Absicht, dass isSignal hier anders erweitert wird als in den channel configs?

File: src/Producers/DiBJetQuantitiesProducer.cc [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+		return diBJetSystem.Phi(); });
+	});
+	LambdaNtupleConsumer<HttTypes>::AddFloatQuantity("diBJetMass", [this](event_type const& event, product_type const& product) {
+		return DiBJetQuantitiesProducer::GetDiBJetQuantity(product, [](RMDLV diBJetSystem) -> double
+	{
+		return diBJetSystem.mass(); });
+	});
+	
+	LambdaNtupleConsumer<HttTypes>::AddFloatQuantity("diBJetDeltaPhi", [](event_type const& event, product_type const& product) {
+		return product.m_diBJetSystemAvailable ? ROOT::Math::VectorUtil::DeltaPhi(product.m_validJets[0]->p4, product.m_validJets[1]->p4) :
```
Comment (swozniewski) [inline src/Producers/DiBJetQuantitiesProducer.cc]: Sollten hier nicht bjets ausgelesen werden?

File: src/Producers/DiBJetQuantitiesProducer.cc [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+		                                        DefaultValues::UndefinedFloat;
+	});
+	LambdaNtupleConsumer<HttTypes>::AddFloatQuantity("diBJetAbsDeltaEta", [](event_type const& event, product_type const& product) {
+		return product.m_diBJetSystemAvailable ? std::abs(product.m_validJets[0]->p4.Eta() - product.m_validJets[1]->p4.Eta()) :
```
Comment (swozniewski) [inline src/Producers/DiBJetQuantitiesProducer.cc]: same here

**Review by swozniewski — CHANGES_REQUESTED** [review]

File: src/Producers/DiBJetQuantitiesProducer.cc (hunk 3 above)
Comment (janekbechtel) [reply src/Producers/DiBJetQuantitiesProducer.cc]: fixed

File: python/data/ArtusConfigs/Run2LegacyAnalysis_base.py (hunk 1 above)
Comment (janekbechtel) [reply python/data/ArtusConfigs/Run2LegacyAnalysis_base.py]: fixed

File: src/Producers/DiBJetQuantitiesProducer.cc (hunk 2 above)
Comment (janekbechtel) [reply src/Producers/DiBJetQuantitiesProducer.cc]: fixed

File: python/data/ArtusConfigs/Run2LegacyAnalysis/2016/et.py [hunk 4]
```diff
@@ -276,7 +278,8 @@ def build_config(nickname, **kwargs):
                                                               "producer:PuppiMetCorrector",
```
Comment (harrypuuter) [inline python/data/ArtusConfigs/Run2LegacyAnalysis/2016/et.py:279]: maybe add the minimalPlotLevelFilter consistently for all three year and channels in the same place in the producer chain ?

**Review by harrypuuter — CHANGES_REQUESTED** [review]

File: python/data/ArtusConfigs/Run2LegacyAnalysis/2016/et.py (hunk 4 above)
Comment (janekbechtel) [reply python/data/ArtusConfigs/Run2LegacyAnalysis/2016/et.py:279]: done. I moved it even higher (after the DecayChannelProducer) as for Run2Legacy, there is no nbtag cut in the MinimalPlotLevelFilter. I tried for 2016 (here reduced_trigger_objects still seems to run and there are no issues.

**Review by harrypuuter — APPROVED** [review]
:+1:

**Review by swozniewski — APPROVED** [review]

**Pull request merged**